src/main/java/com/microsoft/lst_bench/input/Session.java

```diff
@@ -29,4 +31,7 @@
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public interface Session {
   List<Task> getTasks();
+
+  @JsonProperty("connection_manager")
+  @Nullable Integer getConnectionManager();
```

Comment:
[Suggestion] To me connection manager is an internal detail. The framework user should only care about the target cluster where statements will be executed. What do you think about renaming this to target-cluster?

Reply:
I settled for target-endpoint.